### PR TITLE
Make Japanese more natural

### PR DESCRIPTION
**Pull request by tatat**

### CODE REVIEW

File: src/locales/ja-JP.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 const ja_JP = {
     info: {
         title: "ルーナボタン",
-        info: "通知",
+        info: "お知らせ",
```
Comment (tatat) [inline src/locales/ja-JP.js:4]: "情報" might be better than "お知らせ".
Since this doesn't seem to be used currently, I couldn't judge which one is better.

**Comment by monoai** [comment]
Hi there! Thanks for the translation improvements, I checked the changes with the person that's helping me with the JP translations which he is grateful to see. It seems all good, but for _overlapTips_ he suggested "多重再生モードは手動で停止できません。再読み込みで停止して下さい。"

I would like to hear your thoughts about it before we finalize this.

**Comment by tatat** [comment]
Thank you all for reviewing!

> "多重再生モードは手動で停止できません。再読み込みで停止して下さい。"

Definitely! I, too, think that sentence is better!

File: src/locales/ja-JP.js (hunk 1 above)
Comment (monoai) [reply src/locales/ja-JP.js:4]: This is better, though this is probably a remnant of something old that isn't used anymore. Will be judged in the future if it's still needed. Will be resolving this conversation now!

**Comment by monoai** [comment]
Everything looks good! Thank you for the translation improvements! I really appreciate it.

**Pull request merged**